doctr/local.py

```diff
@@ -128,6 +128,12 @@ def GitHub_login(*, username=None, password=None, OTP=None, headers=None):
     if r.status_code == 401:
         two_factor = r.headers.get('X-GitHub-OTP')
         if two_factor:
+            auth_header = base64.urlsafe_b64encode(bytes(username + ':' + password, 'utf8')).decode()
+            login_kwargs = {'auth': None, 'headers': {'Authorization': 'Basic {}'.format(auth_header)}}
```

Comment:
it could potentially but its not really necessary to generate SMS so I just used it like this